ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -71,9 +71,10 @@ SettingsContentBase {
         manageTokensView.saveChanges()
     }
     onSaveChangesClicked: {
+        manageTokensView.saveChanges()
```

Comment:
I'm wondering which is the difference between `onSaveForLaterClicked` and `onSaveChangesClicked` in terms of "saving changes".. For me, the difference should be..  `onSaveChangesClicked` should directly apply the new settings and set as well the order to the `custom one`, otherwise, this save for later doesn't have any sense to me... I think this is a point for future discussion, probably discussion with design team.

Reply:
We discussed this already several times 😁 The current behavior is correct; the "Save for later" option just saves the Custom order but doesn't set it for the main wallet view